integration/run_test.go

```diff
@@ -267,6 +267,16 @@ func TestNetworkRetry(t *testing.T) {
 				expectedSuccess:    false,
 			},
 		},
+		{
+			name: "Permanent network interruption after loading content, success",
```

Comment:
Does this depend on `No network interruption` test case to be ran first? Generally I would prefer test cases to be order agnostic.

Reply:
These tests are each run in their [own container](https://github.com/awslabs/soci-snapshotter/pull/1233/files#diff-8a6db7c58ab40add58f9046cee9ffab883d23e66c3dbc4a315dba43314c06d0bR285), so order doesn't matter.

Reply:
Oh , I understand the permanent network interruption test runs the container to fetch the required spans.

Reply:
Yeah, the container gets run [here](https://github.com/awslabs/soci-snapshotter/pull/1233/files#diff-8a6db7c58ab40add58f9046cee9ffab883d23e66c3dbc4a315dba43314c06d0bR315) for this use case so that when it goes offline it will have the content loaded.